lib/composers/with_redux.js

```diff
@@ -0,0 +1,30 @@
+import compose from '../compose';
+
+export default function composeWithRedux(fn, L, E, options) {
+  const onPropsChange = (props, onData) => {
+    const context = typeof props.context === 'function' ? props.context() : props.context;
```

Comment:
Nope. Loader component will be there. 
Now loader will be the third one and so on.